launcher/mods/archive.py

```diff
@@ -23,6 +23,10 @@ def get_mime_from_file(filename) -> str:
     return 'Unknown'
```

Comment:
Just raise the Exception here instead

Reply:
You are right, at point when it returns Unknown it already failed and there will be exception anyway